pegasus/routes/hoc_routes.rb

```diff
@@ -14,17 +14,17 @@
   JSON.pretty_generate(fetch_hoc_metrics['tutorials'])
 end
 
-# Employee engagement
-get '/api/hour/begin_company/:company' do |company|
-  pass unless DB[:forms].where(kind: 'CompanyProfile', name: company).first
-  pass unless tutorial = DB[:tutorials].where(code: 'codeorg').first
-  launch_tutorial(tutorial, company: company)
-end
```

Comment:
Taking away URLs is scary. Can we leave this in, in case someone has a stale email or cached copy of last year's web page lying around?